src/components/CustomFileInput/index.js

```diff
@@ -36,10 +36,39 @@ export default class CustomFileInput extends Component<Props> {
     event.currentTarget.value = '' // Allow re-uplading the same file
   }
 
+  handleDragOverEvent = (ev: DragEvent) => {
+    // Prevent default behavior (Prevent file from being opened)
+    ev.preventDefault();
+  }
+
+  getFileFromDragEvent = (ev: SyntheticDragEvent<HTMLSpanElement>): ?File => {
+    let dataTransfer: DataTransfer = ev.dataTransfer;
+    let items: DataTransferItemList = dataTransfer.items;
+    let files: FileList = dataTransfer.files;
+
```

Comment:
I think these should be `const` variables instead of `let` 
(bit of Swift happening here? 😉)
Good use of the Flow types btw

Reply:
yepp mixed up a little Swift here 😂 will change to `const`